src/components/pages/StudentsPage/CopyButton.tsx

```diff
@@ -0,0 +1,39 @@
+/** @jsxImportSource @emotion/react */
+
+import { Fab } from '@mui/material';
+import { ContentCopy as ContentCopyIcon } from '@mui/icons-material';
+
+export default function CopyButton({ elementId }) {
+  const copyToClipboard = () => {
+    const temp = document.createElement('div');
+    temp.setAttribute('contentEditable', 'true');
+    temp.innerHTML = document.getElementById(elementId).innerHTML;
+    temp.setAttribute(
+      'onfocus',
+      "document.execCommand('selectAll',false,null)",
+    );
+    document.body.appendChild(temp);
```

Comment:
Let's move the `document.body.appendChild` up.  This way the `'onfocus'` will be immediately followed by the `temp.focus()`. It makes it slightly easier to read and understand the code. 
```suggestion
    document.body.appendChild(temp);
    temp.setAttribute(
      'onfocus',
      "document.execCommand('selectAll',false,null)",
    );
```